autoria.py

```diff
@@ -0,0 +1,242 @@
+
+    No args needed because the list has fixed url.
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Легковые",
+            "value": 1
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories'
+    return make_request(url)
+
+
+def get_bodystyles(category: int) -> json:
+    """Get available bodystyles from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Седан",
+            "value": 3
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories/{}/bodystyles' \
+          .format(str(category))
+    return make_request(url)
+
+
+def get_marks(category: int) -> json:
+    """Get available car marks from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "Acura",
+            "value": 98
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories/{}/marks' \
+          .format(str(category))
+    return make_request(url)
+
+
+def get_models(category: int, mark: int) -> json:
+    """Get available models for selected mark from auto.ria.com.
+
+    Args:
+        category - integer with needed category id.
+        mark - integer with needed mark id
+            (since we're getting models of the mark)
+
+    Returns:
+        JSON-formatted list of pairs ''name: value'' like:
+        [
+            {
+            "name": "CL",
+            "value": 953
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/categories/{}/marks/{}/models' \
+          .format(str(category), str(mark))
+    return make_request(url)
+
+
+def get_states() -> json:
+    """Get available states from auto.ria.com.
+
+    Args aren't needed because the url is fixed.
+
+    Returns:
+        JSON-formatted list of pair ''name: value'' like:
+        [
+            {
+            "name": "Винницкая",
+            "value": 1
+            },
+            ...
+        ]
+    """
+    url = 'http://api.auto.ria.com/states'
+    return make_request(url)
+
+
+def select_item(item_to_select: str, items_list: json) -> int:
+    """Select vehicle type, bodystyle, mark, model from the given list.
+
+    This function is intended to convert human-readable search parameter,
+    for instance, ''Винница'' into API-understandable state identifier,
+    for the given example it would be 1.
+
+    Args:
+        item_to_select - string, could be not 100% accurate as it is in the
+            auto.ria.ua lists, e.g. value ''Харьков'' is acceptable,
+            because the parameter is wildcarted in comprasion like
```

Comment:
Don't really understand what did you mean :flushed: 
Corrected to "wild-carded"...